app/models/concerns/hostext/search.rb

```diff
@@ -92,6 +92,9 @@ module Search
         scoped_search :in => :search_users, :on => :lastname,  :complete_value => true, :only_explicit => true, :rename => :'user.lastname', :operators => ['= ', '~ '], :ext_method => :search_by_user
         scoped_search :in => :search_users, :on => :mail,      :complete_value => true, :only_explicit => true, :rename => :'user.mail',     :operators => ['= ', '~ '], :ext_method => :search_by_user
       end
+
+      private
+      cattr_accessor :fact_values_table_counter
```

Comment:
I'm unsure if this is better or worse than scoped_search's randomness: https://github.com/wvanbergen/scoped_search/blob/v3.2.2/lib/scoped_search/query_builder.rb#L298